src/Tribe/Events/Views/V2/Models/Tickets.php

```diff
@@ -181,13 +181,18 @@ public function fetch_data() {
 
 						$number = number_format_i18n( $stock );
 
+						$ticket_label_singular = tribe_get_ticket_label_singular_lowercase( 'event-tickets' );
+						$ticket_label_plural = tribe_get_ticket_label_plural_lowercase( 'event-tickets' );
+
 						if ( 'rsvp' === $type ) {
-							$text = _n( '%s spot left', '%s spots left', $stock, 'event-tickets' );
+							/* translators: %1$s: Number of stock */
+							$text = _n( '%1$s spot left', '%1$s spots left', $stock, 'event-tickets' );
 						} else {
-							$text = _n( '%s ticket left', '%s tickets left', $stock, 'event-tickets' );
+							/* translators: %1$s: Number of stock, %2$s: Ticket label, %3$s: Tickets label */
+							$text = _n( "%1$s %2$s left", "%1$s %3$s left", $stock, 'event-tickets' );
```

Comment:
* :warning: Mismatched placeholders is probably an error
* :no_entry_sign: The $single arg must not contain interpolated variables. Found "$s".
* :no_entry_sign: The $single arg must not contain interpolated variables. Found "$s".
* :no_entry_sign: The $plural arg must not contain interpolated variables. Found "$s".
* :no_entry_sign: The $plural arg must not contain interpolated variables. Found "$s".

<sub>hash: f8eb98b9fbadf61ac30a81525e489a7f</sub>

Reply:
We should on this case ignore.

Reply:
```suggestion
							$text = _n( "%1$s %2$s left", "%1$s %3$s left", $stock, 'event-tickets' ); // phpcs:ignore
```